lib/mvi/messages/add_person_message.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'socket'
+
+module MVI
+  module Messages
+    # Builds an MVI SOAP XML message for adding a user.
+    #
+    # Call with a user object and use `.to_xml` method to create the XML message
+    #
+    # Example:
+    #  message = MVI::Messages::AddPersonMessage.new(user).to_xml
+    #
+    class AddPersonMessage
+      SCHEMA_FILE_NAME = 'mvi_add_person_template.xml'
+
+      def initialize(user)
+        @user = user
+      end
+
+      def to_xml
+        template = Liquid::Template.parse(
+          File.read(File.join('config', 'mvi_schema', SCHEMA_FILE_NAME))
+        )
+
+        template.render!(build_content(@user))
+      end
+
+      private
+
+      def build_content(user)
+        current_time = Time.current
+        ip_address = Socket.ip_address_list.find { |ip| ip.ipv4? && !ip.ipv4_loopback? }.ip_address
```

Comment:
Curious about the intent here. Obviously our vets-api instances get recycled all the time so in terms of auditability this doesn't help that much. Though not surprising that MVI expects more static IPs

Reply:
Yea, its literally a field thats required for their endpoint. TBH, we could probably fill this in with whatever but I'm not trying to give them a reason to reject us if its just filled with a garbage ip.

Reply:
Silvio, I think your instinct to err on the side of caution and give them the real IP is correct. They may know the CIDR blocks for our VPCs; it's possible that they check that the stated ip is within the CIDR.